_posts/on-working-standing-up-8-years-later.md

```diff
@@ -0,0 +1,53 @@
+
+## Am I still satisfied?
+
+Absolutely yes. I would never no go back. I work standing up most of the day. I sit down only for two reasons:
+
+- To read long chunks of text. For some reason, I'm unable to read for a long time standing up.
+- When I'm too tired. For example, the 3yo disrupted the night.
+
+## Current setup
+
+Recently, my setup changed a bit. Until a year ago, my screens setup was a Full HD portrait screen on the left (23"), a landscape 4K in front of me (28"), and another portrait, Full HD, on the right (23"). I was using the 28" 4K screen at 150% DPI scaling. The only disadvantage of that setup is that using different screens at different DPI scaling creates weird effects when dragging windows across the screens.
+
+I'm now using three 4K screens, all of them at 150% DPI scaling. One portrait on the left (28"), one landscape in the center (32"), and another 32" landscape on the right.
+
+![my desk](/img/posts/on-working-standing-up-8-years-later/desk.jpg){:class="img-fluid mx-auto d-block"}
+
+The angle of the left and right screens is still about 40°. The reasons behind the choice didn't change. To look at the left and right screens at lower grades, I only need to rotate my neck slightly. In the long run, that movement leads to pain and eventually strain. With a higher angle, I rotate my shoulders, and given that I'm standing up, I end up turning most of the body without involving the neck in the movement.
+
+## Ultra-wide curved screens 
+
+I tried, and I failed. I bought a [Samsung C49RG90](https://displaysolutions.samsung.com/monitor/detail/1421/C49RG90) (49") curved screen. It's a great device that doesn't work for me. It's primarily a problem with the curvature radius; it's too low. The long-term effect is the same as having a three-screen setup with about a 15° angle between them. I rotate the neck leading to pain. Not to mention that the screen is so large that to look at the far left and far right sides, I needed to slide left or right due to the limited radius. I returned it after having used it for a day.
+
+## Standing/balance pad
+
+When I'm in my home office, I usually stand on bare feet: no shoes, no socks, no slippers. In the long run, that's not healthy. Our feet are not anymore designed to stand up that long with no supporting device like a proper pair of shoes. About four years ago, thanks to [Daniel Marbach](https://twitter.com/danielmarbach), I bought a balance pad. It's [this one from Airex](https://shop.my-airex.com/Balance-pad-Cloud/207762.1), but wider, I couldn't find the exact model anymore.
+
+The balance pad comes with two advantages and no side effects, as far as I can tell. The first advantage is that it makes standing up on bare feet much more comfortable. The second, and more important, the balance pad makes so that you cannot stand static. It forces you to move constantly, micro-movements, making so that your back muscles are constantly stimulated.
```

Comment:
```suggestion
The balance pad comes with two advantages and no side effects, as far as I can tell. The first advantage is that it makes standing up on bare feet much more comfortable. The second, and more important, is that the balance pad forces you to move constantly, micro-movements, so that your back muscles are constantly stimulated.
```